bonfire_deploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Works. Might be useful to adopt [shellcheck](https://www.shellcheck.net/)'s suggestions.